docs/user_docs/upgrade/upgrade-with-helm/upgrade-kubeblocks-to-0.9.1.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Upgrade to KubeBlocks v0.9.1
+description: Upgrade to KubeBlocks v0.9.1, operation, tips and notes
+keywords: [upgrade, 0.9.1]
+sidebar_position: 1
+sidebar_label: Upgrade to KubeBlocks v0.9.1
+---
+
+# Upgrade to KubeBlocks v0.9.1
+
+:::note
+
+Execute `helm -n kb-system list | grep kubeblocks` to check the current KubeBlocks version you are running before upgrading KubeBlocks.
+
+:::
+
+## Compatibility
+
+KubeBlocks v0.9.1 is compatible with KubeBlocks v0.8 APIs, but compatibility with APIs from versions prior to v0.8 is not guaranteed. If you are using Addons from KubeBlocks v0.7 or earlier (v0.6, etc), DO [upgrade KubeBlocks and all Addons to v0.8 first](./upgrade-kubeblocks-to-0.8.md) to ensure service availability before upgrading to v0.9.
+
+If you are upgrading from v0.8 to v0.9, it's recommended to enable webhook to ensure the availability.
+
+## Upgrade from KubeBlocks v0.9.0
+
+1. Check whether `keepAddons` is set as `true`.
+
+    KubeBlocks v0.8 streamlines the default installed engines. To avoid deleting Addon resources that are already in use during the upgrade, execute the following commands first to check whether `keepAddons` is set as `true`.
+
+    - Check the current KubeBlocks version.
+
+         ```shell
+         helm -n kb-system list | grep kubeblocks
+         ```
+
+    - Set `crd.enabled` as `false` to avoid automatic CRD installation.
+
+         ```shell
+         helm repo add kubeblocks https://apecloud.github.io/helm-charts
+         helm repo update kubeblocks
+         helm -n kb-system upgrade kubeblocks kubeblocks/kubeblocks --version {VERSION} --set crd.enabled=false
```

Comment:
These helm commands are no longer needed. Also, the `keepAddons` option shouldn't be mentioned.

Reply:
> These helm commands are no longer needed. Also, the `keepAddons` option shouldn't be mentioned.

The upgrade steps have been updated, and the related documentation will be synced soon. You can review them later.